packages/ui/Scrollable/useScrollableAdapter.js

```diff
@@ -0,0 +1,51 @@
+import {useRef} from 'react';
+
+const useScrollableAdapter = () => {
```

Comment:
Apdapter's role is store variable into ref.current.
Is this suitable for custom hooks?
I'd like to reduce the number of hooks.